roadmap_page.php

```diff
@@ -96,20 +96,22 @@ function print_version_header( array $p_version_row ) {
 	echo '<div id="' . $t_block_id . '" class="widget-box widget-color-blue2 ' . $t_block_css . '">';
 	echo '<div class="widget-header widget-header-small">';
 	echo '<h4 class="widget-title lighter">';
-	echo '<i class="ace-icon fa fa-road"></i>';
+	print_icon( 'fa-road', 'ace-icon' );
 	echo $t_release_title, lang_get( 'word_separator' );
 	echo '</h4>';
 	echo '<div class="widget-toolbar">';
 	echo '<a data-action="collapse" href="#">';
-	echo '<i class="1 ace-icon fa ' . $t_block_icon . ' bigger-125"></i>';
+	print_icon( $t_block_icon, '1 ace-icon bigger-125' );
 	echo '</a>';
 	echo '</div>';
 	echo '</div>';
 
 	echo '<div class="widget-body">';
 	echo '<div class="widget-toolbox padding-8 clearfix">';
 	if( $t_scheduled_release_date ) {
-		echo '<div class="pull-left"><i class="fa fa-calendar-o fa-lg"> </i> ' . $t_scheduled_release_date . '</div>';
+		echo '<div class="pull-left">';
+		print_icon( 'fa-calendar-o', 'fa-lg' );
+		echo $t_scheduled_release_date . '</div>';
```

Comment:
Should be `echo ' ' . $t_scheduled_release_date . '</div>';`
Now
![grafik](https://user-images.githubusercontent.com/319317/103462628-d6a27880-4d26-11eb-9035-015e7e958f02.png)
Before
![grafik](https://user-images.githubusercontent.com/319317/103462647-f33eb080-4d26-11eb-90dc-6f8262c0afae.png)

Reply:
OK